elastic/kibana
# [Synthetics] - add/edit - address forward ref warnings

## Summary

Addresses forwardRef and other React warnings related to the Add/Edit page.

### Testing

1. Navigate to the Add Page. Open the console. Ensure there are no react warnings in the console related to the add page.
2. Add a monitor
3. Click edit for that monitor
4. Open the console. Ensure there are no react warnings in the console related to the add page.
